lib/ansible/modules/windows/win_package.py

```diff
@@ -26,8 +26,8 @@
     description:
     - Any arguments the installer needs to either install or uninstall the
       package.
-    - If the package is an MSI do not supply the C(/qn), C(/log) or
-      C(/norestart) arguments.
+    - If the package is an MSI do not supply the C(/qn) or C(/norestart) arguments.
+      C(/log) can be provided as an argument.
```

Comment:
I don't think this is true for when it is an MSI. Currently we always add `/L*V temp_log_path` to the list of arguments when calling msiexec https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/windows/win_package.ps1. If multiple `/L` values are present I'm pretty sure msiexec will fail or not behaviour as expected.

There's even a feature request (which I see you've commented on) to keep the logs and/or specifying your own log path for an msi https://github.com/ansible/ansible/issues/38353. I don't want to paint ourselves in the corner and say you can use the log argument and we decide to take that away in the future.

Reply:
I see your point, though I've done quite a few .msi deploys via win_package to Server 2016 and it's been working as expected without showing errors. It may be that the last `/l` item takes effect if there are multiple (I see the code appends `$arguments` after `/L*V"`, but that may or may not be the behaviour for all versions of Windows.

Reply:
If the last argument takes precedence then that's even more of a reason to not explicitly support this. On failures the module will try and read the log and return that for debugging purposes but if the file doesn't exist we could be coming across other issues.